sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -240,6 +241,12 @@ def is_terminated(self, pipeline, is_main_job=True):
       return runner.PipelineState.is_terminal(result.state)
     return True
 
+  def set_cached_source_signature(self, pipeline, signature):
+    self._cached_source_signature[pipeline] = signature
+
+  def cached_source_signature(self, pipeline):
```

Comment:
nit; how about `get_cached_source_signature` instead of `cached_source_signature`. That would be more parallel with the set_ and distinguish from the _cached_source_signature dictionary.

Reply:
Renamed the function.